test/e2e/storage/testsuites/volumemode.go

```diff
@@ -60,6 +60,9 @@ func (t *volumeModeTestSuite) getTestSuiteInfo() TestSuiteInfo {
 	return t.tsInfo
 }
 
+func (t *volumeModeTestSuite) skipUnsupportedSuite(driver TestDriver, pattern testpatterns.TestPattern) {
+}
```

Comment:
I think we don't need to test both pre-provisioned and dynamic volumes, they should behave the same in this suite.

Reply:
@mkimuram commented [here](https://github.com/kubernetes/kubernetes/pull/81375#issuecomment-521041120) that the failure handling is different between preprovisioned and dynamic

Reply:
ack